modules/migration-known-issues.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-known-issues_{context}']
+= Known issues
+
+This release has the following known issues:
+
+* During migration, the CAM tool preserves the following namespace annotations:
+
+** `openshift.io/sa.scc.mcs`
+** `openshift.io/sa.scc.supplemental-groups`
+** `openshift.io/sa.scc.uid-range`
++
+These annotations preserve the UID range, ensuring that the containers retain their file system permissions on the target cluster. There is a risk that the migrated UIDs could duplicate UIDs within an existing or future namespace on the target cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1748440[*BZ#1748440*])
+
+* Restic times out when migrating more than 50 projects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1749831[*BZ#1749831*])
```

Comment:
I would remove the 50 projects and 500 pods from the "known issues" section.   
Instead, we need a new section describing the default limit in the tool "10 namespaces" and "100 pods".   And explain that those limits can be change using a the following procedure : @jwmatthews to provide.